libkpod/container_server.go

```diff
@@ -590,6 +597,46 @@ func (c *ContainerServer) ListContainers() []*oci.Container {
 	return c.state.containers.List()
 }
 
+// ListRunningContainers returns a list of all containers that are currently running
+func (c *ContainerServer) ListRunningContainers() []*oci.Container {
```

Comment:
Can we do a more generic ListContainers() that accepts filter functions to determine what's being outputted?